### PR TITLE
Add standalone installers to Ruff installation in README

### PR DESCRIPTION
**Pull request by zanieb**
Note this is already included in our installation page at `docs/installation.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,10 +116,28 @@ For more, see the [documentation](https://docs.astral.sh/ruff/).
...
+```
+
+Or, from [PyPI](https://pypi.org/project/ruff/):
+
+```shell
+# With pip.
+pip install uv
+
+# With pipx.
+pipx install uv
```
Comment (dhruvmanila) [inline README.md:140]:
```suggestion
# With pip.
pip install ruff

# With pipx.
pipx install ruff
```

**Review by dhruvmanila — APPROVED** [review]
Do we want to keep the same order as in `docs/installation.md`?

**Comment by zanieb** [comment]
I'm not sure, I took the order from uv @charliermarsh

**Comment by charliermarsh** [comment]
Sorry, I missed this, but I would say they go below PyPI. I can change it if you want.

**Pull request merged**